zmq-sys/Cargo.toml

```
@@ -14,9 +14,11 @@ build = "build/main.rs"
 links = "zmq"
 
 [features]
+libsodium = ['libsodium-sys-stable']
```

```suggestion
libsodium = ['dep:libsodium-sys-stable']
```
And shouldn't these be double quotes?